core-api/src/main/java/com/sequenceiq/cloudbreak/doc/ModelDescriptions.java

```diff
@@ -260,6 +260,7 @@ private ModelDescriptions() {
         public static final String CLUSTER_ATTRIBUTES = "Additional information for ambari cluster";
         public static final String RDSCONFIG_IDS = "RDS configuration ids for the cluster";
         public static final String RDSCONFIG_NAMES = "RDS configuration names for the cluster";
+        public static final String READBEAMS_DB_SERVER_CRN = "Contains valid Crn for a readbeams database server";
```

Comment:
The spelling should be "REDBEAMS", not "READBEAMS". Although the past tense of "read" is pronounced like "red" 😄